src/oauth.pinterest.js

```diff
@@ -0,0 +1,85 @@
+(function () {
+  'use strict';
+
+  angular.module('oauth.pinterest', ['oauth.utils'])
+    .factory('$ngCordovaPinterest', pinterest);
+
+  function pinterest($q, $http, $cordovaOauthUtility) {
+    return {
+      signin: oauthPinterest
+    };
+
+    /*
+     * Sign into the Pinterest service
+     *
+     * @param    string clientId
+     * @param    string clientSecret
+     * @param    array appScope
+     * @param    object options
+     * @return   promise
+     */
+    function oauthPinterest(clientId, clientSecret, appScope, options) {
+      var deferred = $q.defer();
+      if (window.cordova) {
+        if ($cordovaOauthUtility.isInAppBrowserInstalled()) {
+          var redirect_uri = "http://localhost/callback";
+          if (options !== undefined) {
+            if (options.hasOwnProperty("redirect_uri")) {
+              redirect_uri = options.redirect_uri;
+            }
+          }
+          var flowUrlParam = $cordovaOauthUtility.generateUrlParameters({
+            client_id: clientId,
+            redirect_uri: redirect_uri,
+            scope: appScope.join(","),
```

Comment:
Is this correct? The documentation said that the scope is separeted by `,` on URL request?